### PR TITLE
Add query dump test for edge indices

### PR DESCRIPTION
**Pull request by Josipmrden**
Unit test for query dump for edge indices

### CODE REVIEW

**Comment by Josipmrden** [comment]
### Standard development
- [X] Update unit/E2E tests
- [X] Compare the [benchmarking results](https://bench-graph.memgraph.com/) between the master branch this branch


### CI Testing Labels
- [X] Select the appropriate CI test labels _(CI -build=**build-name** -test=**test-suite**)_


### Documentation checklist -> not needed, just a test

File: tests/unit/query_dump.cpp [hunk 1]
```diff
@@ -682,6 +682,52 @@ TYPED_TEST(DumpTest, IndicesKeys) {
   }
 }
 
+// NOLINTNEXTLINE(hicpp-special-member-functions)
+TYPED_TEST(DumpTest, EdgeIndicesKeys) {
+  if (this->config.salient.storage_mode == memgraph::storage::StorageMode::ON_DISK_TRANSACTIONAL) {
+    GTEST_SKIP() << "Point index not implemented for edge indices";
```
Comment (Ignition) [inline tests/unit/query_dump.cpp:688]: Nearly, but not quite

**Review by Ignition — APPROVED** [review]
